src/abyssal_tome/app.py

```
@@ -1,729 +1,88 @@
+        yield db
+    finally:
+        db.close()
+
+# --- API Endpoints ---
+@app.get("/rulings", response_model=List[Ruling])
+def get_rulings(skip: int = 0, limit: int = 100, db: Session = Depends(get_db)):
+    rulings = db.query(RulingDB).offset(skip).limit(limit).all()
+    return rulings
+
+@app.get("/rulings/{ruling_id}", response_model=Ruling)
+def get_ruling(ruling_id: str, db: Session = Depends(get_db)):
```

Ruff (0.11.9)

Missing return type annotation for public function `get_rulings`

(ANN201)